barcode/barcode_api.py

```diff
@@ -0,0 +1,30 @@
+from uber.api import fields as attendee_fields
```

Comment:
on that note, I actually would have loved to just have this function call the same function as the attendee.lookup() API and not deal directly with doing the DB query and messing with fields.

as far as I'm concerned, barcode just gets you a bdage#, and the existing API should take over from that part